src/vue/backend/src/components/listingsCard.vue

```diff
@@ -11,7 +11,7 @@
                 v-bind="listing"
             >
             <template v-slot:propertyCardImage>
-                    <img class="property-card__image" :src="listing.image">
+                    <img class="property-card__image" :src="listing.image" alt="listing image">
```

Comment:
For listing image, we can have it leverage the listings address. Something like
`:alt="`Image of ${listing.address}`"
that way the alt tag describes the image better